react/components/Share/utils/socialResolver.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import * as ReactShare from 'react-share'
+
+import { SOCIAL_ENUM_TO_COMPONENT } from '../constants/social'
+
+export function SocialButton({ url, size, socialEnum }) {
```

Comment:
why not `export default`?

Reply:
why?

Reply:
I used this so I can export the component but also, in the future, export social resolvers functions. That's why the file name is this one.

Reply:
And I created this component to help to selecting the social button. To divide this logic.